sputnikdao2/src/proposals.rs

```diff
@@ -164,6 +171,7 @@ impl From<Action> for Vote {
 
 /// Proposal that are sent to this DAO.
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize)]
+#[borsh(crate = "near_sdk::borsh")]
 #[cfg_attr(not(target_arch = "wasm32"), derive(Debug))]
 #[serde(crate = "near_sdk::serde")]
```

Comment:
Please, refactor all manual derives of borsh and serde Serialize/Deserialize and `crate` specifications with just [`#[near(serializers = [json, borsh])]`](https://docs.rs/near-sdk/latest/near_sdk/attr.near.html#nearserializers-annotates-structsenums), not only it will be shorter, it will also correctly enable ABI, and we will be able to use `cargo near build` without disabling ABI.

Reply:
@frol I'm not sure how to handle situations when there are parameters to the serializer, such as `#[serde(crate = "near_sdk::serde", untagged)]`. So I have not touched these.

Reply:
@petersalomonsen skip the crate part and only live `#[serde(untagged)]`